resources_pages/install_ds_stack_mac.md

```diff
@@ -120,17 +125,38 @@ git version 2.24.2 (Apple Git-127)
 
 > Note: If you run into trouble, please see that Install Git > Mac OS section from [Happy Git and GitHub for the useR](http://happygitwithr.com/install-git.html#mac-os) for additional help or strategies for Git installation.
 
-Next, we need to configure Git by telling it who you are, your email, and setting the default text editor to VS Code. To do this type the following into the terminal (replacing Jane Doe and janedoe@example.com, with your name and email, respectively):
+### Configuring Git user info
+
+Next, we need to configure Git by telling it your name and email. To do this type the following into the terminal (replacing Jane Doe and janedoe@example.com, with your name and email (the same used to sign up for GitHub), respectively):
```

Comment:
Should they use their Github.com or ubc.github.ca email as the default?

Reply:
It doesn't matter, I don't think Git will know the difference, it just takes one email. TBH, I don't think it even has to be an email associated with the GitHub accounts. I think it's just a recommendation about it matching up to the one they use with GitHub. You can associate multiple emails with GitHub, and I think even associate them with different orgs.